parser/src/parser.rs

```diff
@@ -453,13 +458,15 @@ where
     fn parse_class_member(&mut self) -> ParserResult<'f, ast::ClassMember<'f>> {
         spanned!(self, {
             let mut attrs = Vec::new();
-            while self
-                .eat_optional(Exactly(TokenKind::Operator(Operator::Slash)))
-                .is_some()
-            {
-                self.eat(Exactly(TokenKind::Operator(Operator::Slash)))?;
-                self.eat(Exactly(TokenKind::Operator(Operator::QuestionMark)))?;
-                attrs.push(self.parse_attribute()?);
+            if std::env::var("VANILLA").is_err() {
```

Comment:
- env var name into a global constant instead of distributing it 3 times over the code base???
- parse_class_member is called several times, caching the lookup?

You can copy paste from here: https://github.com/comprakt/comprakt/commit/7d3708a1d89d3d07618b5cd4c6a096f6e214eb2e